frontend/.env-example

```diff
@@ -13,4 +13,4 @@ NUXT_PUBLIC_MAINTENANCE_TS=1717700652
 NUXT_PUBLIC_SHOW_IN_DEVELOPMENT=
 NUXT_PUBLIC_STRIPE_BASE_URL=
 NUXT_PUBLIC_V1_DOMAIN=
-PRIVATE_SSR_SECRET=
+NUXT_PRIVATE_SSR_SECRET=
```

Comment:
np: commit msg

chore(env): prefix variable with `NUXT`

So that the `environment variable` could also be used locally